Makefile

```diff
@@ -1,6 +1,9 @@
 SHELL = /bin/bash
 
-config:
+ninja/ninja:
+	./scripts/buildNinjaBinary.js
+	
+config: ninja/ninja
```

Comment:
Not sure why this is part of the Makefile.

The core 2 instructions in CONTRIBUTING are ninja config + build -- does that still work?

Reply:
Yes! But these 2 core instructions need a ninja binary first. So before running "ninja.js config", we need to make sure that the ninja binary was built.

Reply:
Hmm, or maybe this should not be part of the Makefile, as the ninja build should happen on npm install already. Will look into it.

Reply:
Fixed ninja compilation on `npm install`. No change to the Makefile is needed.